### PR TITLE
Update KinderGartenGardenTest.cs

### PR DESCRIPTION
**Pull request by Rich-Hopkins**
Corrected incorrect student names in Surprise_garden asserts.

### CODE REVIEW

File: exercises/kindergarten-garden/KinderGartenGardenTest.cs [hunk 1]
```diff
@@ -57,9 +57,9 @@ public void Full_garden()
     public void Surprise_garden()
     {
         var garden = new Garden(new [] { "Samantha", "Patricia", "Xander", "Roger" }, "VCRRGVRG\nRVGCCGCV");
-        Assert.That(garden.GetPlants("Patricia"), Is.EqualTo(new[] { Plant.Violets, Plant.Clover, Plant.Radishes, Plant.Violets }));
-        Assert.That(garden.GetPlants("Roger"), Is.EqualTo(new[] { Plant.Radishes, Plant.Radishes, Plant.Grass, Plant.Clover }));
-        Assert.That(garden.GetPlants("Samantha"), Is.EqualTo(new[] { Plant.Grass, Plant.Violets, Plant.Clover, Plant.Grass }));
-        Assert.That(garden.GetPlants("Xander"), Is.EqualTo(new[] { Plant.Radishes, Plant.Grass, Plant.Clover, Plant.Violets }));
+        Assert.That(garden.GetPlants("Samantha"), Is.EqualTo(new[] { Plant.Violets, Plant.Clover, Plant.Radishes, Plant.Violets }));
```
Comment (Rich-Hopkins) [inline exercises/kindergarten-garden/KinderGartenGardenTest.cs]: Student names in the asserts were associated with the wrong portions of the passed parameter.

**Comment by ErikSchierboom** [comment]
Sorry for the late reply. I'll try and look at this tomorrow.

**Comment by ErikSchierboom** [comment]
Actually, this is not a bug. If you read the [exercise's readme](https://github.com/exercism/x-common/blob/master/exercises/kindergarten-garden/description.md), you'll find the following line: "Each child gets 4 cups, two on each row. The children are assigned to cups in **alphabetical** order." I've bolded the alphabetical part, which is probably what you have missed reading the exercise. 

If you have the students  "Samantha", "Patricia", "Xander" and "Roger", they are assigned cups in the order:  "Patricia", "Roger", "Samantha", "Xander". Hence, the current test and default implementation are correct.

**Comment by Rich-Hopkins** [comment]
Ah, I see it now. Thanks!

**Comment by ErikSchierboom** [comment]
You're welcome! And thanks that you were willing to go to the effort of submitting a PR to fix any problem. It's much appreciated!

**Pull request closed without merging**